charts/prefect-server/values.yaml

```diff
@@ -241,6 +241,23 @@ backgroundServices:
   # ref: https://docs.prefect.io/v3/develop/settings-ref#logging-level-2
   # -- sets PREFECT_LOGGING_SERVER_LEVEL
   loggingLevel: WARNING
+  
+  # Autoscaling configuration
+  # requests MUST be specified if using an HPA, otherwise the HPA will not know when to trigger a scale event
+  autoscaling:
```

Comment:
I might just be missing it, but I think we'll need an HPA template under `templates/background-services/` right?

Reply:
I was playing around with HPA for background services and forgot to remove this section.

Doesn't seem right to have autoscaling because there should only be one instance of this running, correct @mitchnielsen?